test/algorithms/excited_state_solvers/test_excited_states_solvers.py

```diff
@@ -21,7 +21,11 @@
 
 from qiskit_nature.drivers import UnitsType
 from qiskit_nature.drivers.second_quantization import PySCFDriver
-from qiskit_nature.mappers.second_quantization import JordanWignerMapper
+from qiskit_nature.mappers.second_quantization import (
+    BravyiKitaevMapper,
+    JordanWignerMapper,
+    ParityMapper,
```

Comment:
Seeing these, did you also test the BKSF mapper at least once locally?

Reply:
I did not try with BKSF. That is a bit different since it generates mappings based on interactions and quickly becomes less than viable at present due to the much larger resultant number of qubits, and I generally never use that. In this case for H2 its small enough to be viable but I did not try it.

Reply:
Fair enough then :+1: